test/config-next/ca.json

```diff
@@ -4,6 +4,12 @@
     "rsaProfile": "rsaEE",
     "ecdsaProfile": "ecdsaEE",
     "debugAddr": "localhost:8001",
+    "grpc": {
```

Comment:
Can you refresh my memory: for the other components that have been converted to `grpc` versions, did we use the presence of a `grpc` config stanza as the feature flag, or was there an explicit `"UseXXXWithGRPC": "true"`?

I'm not picky about one over the other I just want to make sure we do it consistently so that someone doesn't add the `grpc` stanza to a config thinking "I'll add this ahead of time for when we enable GRPC for the XXXX" not realizing they have done so by adding the config.